projects/plugins/jetpack/modules/likes/style.css

```diff
@@ -136,11 +136,23 @@ div.sd-box {
 	border-top: 1px solid rgba(0,0,0,.13);
 }
 
+
+.jetpack-likes-widget-unloaded .likes-widget-placeholder,
+.jetpack-likes-widget-loading .likes-widget-placeholder,
+.jetpack-likes-widget-loaded iframe {
+	display: block;
+}
+
+.jetpack-likes-widget-loaded .likes-widget-placeholder,
+.jetpack-likes-widget-unloaded iframe,
+.jetpack-likes-widget-loading iframe {
+	display: none;
+}
+
 .entry-content .post-likes-widget, .post-likes-widget,
 .comment-likes-widget {
 	margin: 0;
 	border-width: 0;
-	display: block;
```

Comment:
This `display: block` is IMO quite important, although removing it maybe fixes part of the layout shift. Without this, the `iframe` will have `display: inline` by default. But visually it's a block. Switching between `inline` and `block` changes layout (vertical spacing) because of how `line-height` works and how inline content is aligned to the baseline.

Reply:
This is removed from here because we add the same rule [above](https://github.com/Automattic/jetpack/pull/42361/files#diff-fda7012c269c04c6e9c61cda097274dedf76a96b5fc1861bfc97fa48d95e9738R142).

And with this [one](https://github.com/Automattic/jetpack/pull/42361/files#diff-fda7012c269c04c6e9c61cda097274dedf76a96b5fc1861bfc97fa48d95e9738R148), the change is between `display:none` to `display:block`.

I removed it from here because:
1. I added above as I mentioned
2. I'd have to add more css specificity to override this rule, which I think it's not needed.

Am I missing something? 🤔

Reply:
Thanks for the clarification, my concern was that `display:block` gets completely removed, but that turns out not to be true. :+1: